### DISCUSSION TITLE
Repository vs. Organization level projects in the New Projecst beta?

### DISCUSSION THREAD

**Discussion by abalter** [community/community#52846] [Projects and Issues]
### Select Topic Area

Product Feedback

### Body

# Topic: Product Feedback

In most ways, the new Projects beta seems to only improve on the "classic." However, there are two properties that seems like downgrades. I wonder if these are bugs, features, or just not-gotten-to-yet?

1. ALL repository projects are linked to the organization (if they are under an organization). I'm not sure that when I click on Projects under an organization I want to see ALL projects in all repositories.
2. There is  no longer a link at the project back to the repository or even the organization!
3. As a result, we are no longer able to create a project FOR the organization.

## New Projects: Organization
![image](https://user-images.githubusercontent.com/5349876/232118303-e0adae35-253d-4bb5-ac64-ee2fa1b465ae.png)

###  At project
![image](https://user-images.githubusercontent.com/5349876/232119827-8565cfc0-f586-4975-9f8c-33ae42c2e658.png)

### At organization
![image](https://user-images.githubusercontent.com/5349876/232119958-932c3316-052d-47ef-9f62-f547b99d347c.png)

## New Projects at Repository/Project
No links at all

![image](https://user-images.githubusercontent.com/5349876/232120357-5b833306-70a9-438f-b0f5-50efde89fcf3.png)

## Classic Projects: At Repository
Has links
![image](https://user-images.githubusercontent.com/5349876/232119235-4fe09396-8107-48ae-b904-888a3722f014.png)

## Note 1
I followed [the documentation](https://docs.github.com/en/issues/planning-and-tracking-with-projects/creating-projects/creating-a-project) for creating projects under organization or as a user.

## Note 2
There are a number of historical questions on StackOverflow of people asking _how to create separate organization vs repository level projecst_, and received the answer about how it used to work. This suggests to me that it is a desirable feature.

**Comment by CodeMaster7000** [comment]
Hi @abalter,

I also think that it is most likely a feature. You can see how it was possible on [GitHub Docs](https://docs.github.com/en/issues/organizing-your-work-with-project-boards/managing-project-boards/creating-a-project-board), but I am pretty sure that these just need to be updated.

**Reply by abalter** [reply]
Thanks for chiming in! I hope that someone from github will chime in and that you are correct.
